Backpack/StarRating/Classes/BPKStarRating.m

```diff
@@ -0,0 +1,147 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKStarRating.h"
+
+#import <Backpack/Common.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface BPKStarRating ()
+
+@property (nonatomic) UIStackView *stackView;
+@property (nonatomic) NSArray<BPKStar *> *stars;
+@property (nonatomic) NSUInteger maxRating;
```

Comment:
This looks like a constant so could we make it 

```objective-c
const NSUInteger MAX_RATING = 5;
```

instead?

Reply:
Done